Library/Formula/shibboleth-sp.rb

```diff
@@ -0,0 +1,107 @@
+class ShibbolethSp < Formula
+  desc "Shibboleth 2 Service Provider daemon"
+  homepage "https://wiki.shibboleth.net/confluence/display/SHIB2"
+  url "http://shibboleth.net/downloads/service-provider/latest/shibboleth-sp-2.5.5.tar.gz"
+  sha256 "30da36e0bba2ce4606a9effc37c05cd110dafdd6d3141468c4aa0f57ce4d96ce"
+
+  depends_on "curl" => "with-openssl"
+  depends_on "opensaml"
+  depends_on "xml-tooling-c" => "with-openssl"
+  depends_on "xerces-c"
+  depends_on "xml-security-c"
+  depends_on "log4shib"
+  depends_on "boost"
+  depends_on "homebrew/apache/httpd22" => :optional
+  depends_on "homebrew/apache/httpd24" => :optional
```

Comment:
Got rid of the depends_on and replaced with an option to build 2.2 mod_shib lib.